src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -212,7 +219,13 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({
     quickPinchZoomInnerRef.current?.style.setProperty('transform', transformValue)
   }
 
-  const isZoomedIn = scale > 1.005
+  const clearSelectedLayer = () => {
+    if (selectedLayer?.current) {
+      setSelectedLayer(undefined)
+    }
+  }
+
+  const isZoomedIn = scale > 1.07
```

Comment:
If it's not a problem can you add a comment what 1.07 means and why it's correct? just no idea why 1.005 is wrong and 1.07 fixes something.